tests/testSeparableXYTransform.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2
+#
+# LSST Data Management System
+# Copyright 2015 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""
+Tests for lsst.afw.geom.SeparableXYTransform and Functor classes.
+"""
+import unittest
+import numpy as np
+import numpy.random as random
+import lsst.utils.tests
+import lsst.afw.geom as afwGeom
+
+def num_deriv(func, x, eps=1e-7):
+    h = eps*max(abs(x), 1)
+    xp = x + h
+    dx = xp - x
+    return (func(x + dx) - func(x))/dx
+
+class FunctorTestCase(unittest.TestCase):
+    def setUp(self):
+        self.pars = [('', 5, 30), ('', 2, 30, 4000)]
+        self.funcs = [afwGeom.LinearFunctor,
+                      afwGeom.EdgeRolloffFunctor]
+        self.xvals = np.logspace(-8, 1, 10)
+    def tearDown(self):
+        pass
+    def testDerivs(self):
+        funcs = [func(*pars) for func, pars in zip(self.funcs, self.pars)]
+        for func in funcs:
+            for xx in self.xvals:
+                self.assertAlmostEqual(func.deriv(xx), num_deriv(func, xx),
+                                       places=6)
+
+class SeparableXYTransformTestCase(unittest.TestCase):
+    def setUp(self):
+        random.seed(48091)
+        self.xpars = 'SeparableXYTransform', 2, 30, 4096
+        self.ypars = 'SeparableXYTransform', 2, 30, 4004
+        # Generate test points, drawing randomly from within 2 pixels
+        # around edges of sensors
+        dxy = 2.
+        npts = 10
+        self.xvals = np.concatenate((random.uniform(0, dxy, size=npts),
+                                     random.uniform(self.xpars[-1] - dxy,
+                                                    self.xpars[-1], size=npts)))
+        self.yvals = np.concatenate((random.uniform(0, dxy, size=npts),
+                                     random.uniform(self.ypars[-1] - dxy,
+                                                    self.ypars[-1], size=npts)))
+        self.functorClass = afwGeom.EdgeRolloffFunctor
+    def tearDown(self):
+        pass
```

Comment:
That's not necessary unless the objects in `setUp` are `Citizen`s, in which case it looks like there's a memory leak.